docs/source/configuration/settings-reference.md

```diff
@@ -10,17 +10,17 @@ This list is still incomplete, contributions are welcomed!
 ### navDepth
 
 !!! block ""
-    Navigation levels depth used in the navigation endpoint calls. Increasing this is useful for implementing fat navigation menus. Defaults to `1`.
+Navigation levels depth used in the navigation endpoint calls. Increasing this is useful for implementing fat navigation menus. Defaults to `1`.
 
 ### defaultBlockType
 
 !!! block ""
-    The default block type in Volto is "text", which uses the current DraftJS-based implementation for the rich text editor. Future alternative rich text editors will need to use this setting and replace it with their block type. The block definition should also include the `blockHasValue` function, which is needed to activate the Block Chooser functionality. See this function signature in [Blocks > Settings](../blocks/settings.md).
+The default block type in Volto is "text", which uses the current DraftJS-based implementation for the rich text editor. Future alternative rich text editors will need to use this setting and replace it with their block type. The block definition should also include the `blockHasValue` function, which is needed to activate the Block Chooser functionality. See this function signature in [Blocks > Settings](../blocks/settings.md).
```

Comment:
@tiberiuichim @giuliaghisini This is due to a prettier or a linter? I think it should be indented, isn't it?

Reply:
AFAIK it should be indented